src/components/Fruit.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function Fruit( { name, state, setState } ) {
+
+    return (
+        <article className={state === 0 ? "fruitmand-leeg" : "fruitmand-vol"}>
+            <div><span className="fruit">{name}</span>
+                <button type="button" onClick={() => {state === 0 ? setState(0) : setState(state - 1)}}>-
```

Comment:
Het is wat netter om de button door middel van de `disabled` property uit te zetten
```suggestion
                <button type="button" disabled={state === 0} onClick={() => setState(state - 1)}>-
```